notifier/notifier.go

```diff
@@ -540,24 +560,28 @@ func newAlertmanagerSet(cfg *config.AlertmanagerConfig, logger log.Logger) (*ale
 // sync extracts a deduplicated set of Alertmanager endpoints from a list
 // of target groups definitions.
 func (s *alertmanagerSet) sync(tgs []*targetgroup.Group) {
-	all := []alertmanager{}
+	allAms := []alertmanager{}
+	allDams := []alertmanager{}
```

Comment:
It's not obvious from context what "Dams" are, the naming could be improved

Reply:
Would "DroppedAms" work?

Reply:
Yes